aspnetcore/blazor/debug.md

```diff
@@ -221,6 +221,21 @@ In the preceding example, `MyHostedApp.Server.dll` is the *Server* app's assembl
 
 Blazor provides a debugging proxy that implements the [Chrome DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/) and augments the protocol with .NET-specific information. When debugging keyboard shortcut is pressed, Blazor points the Chrome DevTools at the proxy. The proxy connects to the browser window you're seeking to debug (hence the need to enable remote debugging).
 
+## Breakpoints in `OnInitialized{Async}` not hit
+
+The Blazor framework's debugging proxy takes a short time to launch, so breakpoints in the [`OnInitialized{Async}` lifecycle method](xref:blazor/components/lifecycle#component-initialization-methods) might not be hit. We recommend adding a delay at the start of the method body to give the debug proxy some time to launch before the breakpoint is hit. You can include the delay based on an [`if` compiler directive](/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if) to ensure that the delay isn't present for a release build of the app:
+
+```csharp
+protected override void OnInitialized()
+{
+#if DEBUG
+    xxxx
```

Comment:
For async methods, users can use `await Task.Delay(xxxx)`. For sync methods, users can use `Thread.Sleep(xxxx)`.